### ISSUE TITLE
Use Renovate on GitLab

### ISSUE THREAD

**Issue by simonschaufi** [FriendsOfTYPO3/tea#1141]
Because dependabot is not available on gitlab, Renovate should be used on GitLab instances:

Here is a great integration config: https://github.com/pagemachine/renovate-config

**Comment by lukaszuznanski** [comment]
There is actually Renovate bot on Gitlab https://git.typo3.org/renovate, we can use it.

**Comment by simonschaufi** [comment]
Yes but this bot hadn't run for 2 months. I don't think it's active.
